### ISSUE TITLE
Full-screen button on Liluo.io does not work on iPhone

### ISSUE THREAD

**Issue by tristanbob** [4ian/GDevelop#3917]
## Describe the bug

Games hosted on Liluo have a button to open the game in full-screen mode.  This button does not work on iPhone.

## To Reproduce

1. Go to https://liluo.io/ on an iPhone
2. Click on full-screen button icon
3. Nothing happens

## Other details

* iPhone 12, bug present on both Safari and Chrome

**Comment by 4ian** [comment]
Can you add the version of iOS you're running on?

**Comment by tristanbob** [comment]
> Can you add the version of iOS you're running on?

IOS 15.5 (19F77)

**Comment by Mario655-tech** [comment]
Can confirm too on iOS 15.5,

**Comment by 4ian** [comment]
Right, turns out I never realised but fullscreen is simply... not supported on Safari on iPhones 🤦🏻‍♂️ Only iPads.
We'll have to do something else to give maximum screen estate to games.
